angular/src/index.ts

```diff
@@ -50,7 +50,6 @@ export {
   createGesture,
   iosTransitionAnimation,
   mdTransitionAnimation,
-  IonicSwiper,
   IonicSlides,
```

Comment:
`IonicSlides` is still being exported from the Angular library entry point. Should that be removed with this? If so, should the const also be removed from `@ionic/core`?